Bug 1882097: Fix layout issue which appears only on Safari

jerolimov opened the pull request:
**Fixes**: 
https://issues.redhat.com/browse/ODC-4910
https://bugzilla.redhat.com/show_bug.cgi?id=1882097

**Analysis / Root cause**: 
There are `position:` issues which only appears in Safari when viewing and editing a pipeline.

**Solution Description**: 
Remove and add a position css rule so that the pipeline visualization looks similar in Safari and the others browsers. Tested this in latest Chrome, Firefox and Safari.

**Screen shots / Gifs for design review**: 
Chrome 85, Firefox 80 and Safari 14 on macOS:
![odc-4910 mov](https://user-images.githubusercontent.com/139310/94082450-45576800-fe01-11ea-8980-2a9e6080157a.gif)

**Unit test coverage report**: 
Unchanged

**Test setup:**
* Test a Pipeline visualization in read only mode (open an existing pipeline)
* Create a new pipeline and check if the Select task was also shown at the right position

**Browser conformance**: 
- [x] Chrome
- [x] Firefox
- [x] Safari
- [ ] Edge

jerolimov commented:
/bugzilla refresh

jerolimov commented:
/kind bug
/bugzilla refresh

christianvogt commented:
Small side effect of using `position: fixed` is that the popover now overlaps the entire button because the node no longer has a height.
![image](https://user-images.githubusercontent.com/14068621/94059507-85dac580-fdb0-11ea-8b72-9143ad153c35.png)

jerolimov commented:
> Small side effect of using `position: fixed` is that the popover now overlaps the entire button because the node no longer has a height.

Thanks for finding this. I need to add an additional height which we already use for the svg part. Updated the video above again.

I noticed that there are some more small issues when using the browser zoom, esp. on Safari.

andrewballantyne reviewed (approved):
Tested it on Safari, FF, Chrome, and Edge (Chromium) and it works as expected.

christianvogt commented:
/lgtm

jerolimov commented:
/cherry-pick release 4.5

jerolimov commented:
/cherry-pick release-4.5
